### PR TITLE
Connect port state manager to Forch orchestration components

### PR DESCRIPTION
**Pull request by didovesei**

### CODE REVIEW

File: forch/port_state_manager.py [hunk 1]
```diff
@@ -72,32 +78,69 @@ def _handle_authenticated_state(self):
...
-    def handle_authenticated_device(self, mac):
-        """initialize or update the state machine for an authenticated device"""
+        isolation_behavior = device_behavior.isolation_behavior
+        if isolation_behavior:
+            self._static_port_behaviors[mac] = isolation_behavior
+
+        if device_behavior.segment:
+            self.handle_auth_result(mac, device_behavior, static=True)
+
+    def handle_auth_result(self, mac, device_behavior, static=False):
```
Comment (grafnu) [inline forch/port_state_manager.py:109]: what about this says it's an auth result?  What happens if the device_behavior comes from something other than auth?  (My point is your function name is overly narrow)

File: forch/faucetizer.py [hunk 2]
```diff
@@ -18,6 +18,10 @@
 
 INCLUDE_FILE_SUFFIX = '_augmented'
 TESTING_PORT_IDENTIFIER_DEFAULT = 'TESTING'
+BEHAVIOR = 'behavior'
```
Comment (grafnu) [inline forch/faucetizer.py:21]: these are overly generic for top-level constants, so should be qualified somehow... e.g. so it reads DEVICE.STATIC or DEVICE_STATIC

File: forch/port_state_manager.py (hunk 1 above)
Comment (didovesei) [reply forch/port_state_manager.py]: Changed to handle_device_behavior()

File: forch/faucetizer.py (hunk 2 above)
Comment (didovesei) [reply forch/faucetizer.py]: Updated. PTAL..

**Review by grafnu — APPROVED** [review]

**Pull request merged**